core/templates/pages/welcome-guide-page/welcome-guide-page.directive.html

```diff
@@ -0,0 +1,79 @@
+<div style="height: auto; background-color: #FFFFFF;" class="row content">
+  <div  class="col-lg-7">
+    <div class="oppia-welcome-guide-header">
+      <h1 class="oppia-welcome-guide-h1" translate="I18N_WELCOME_GUIDE_TITLE"> </h1>
+      <p class = "oppia-welcome-guide-p" translate="I18N_WELCOME_GUIDE_PARAGRAH_1">
+      </p>
+
+    </div>
+  </div>
+  <div class="oppia-welcome-guide-img col-lg-5">
+    <picture>
+      <source type="image/webp" ng-srcset="<[$ctrl.headerImgUrlWebp]>">
+      <source type="image/jpeg" ng-srcset="<[$ctrl.headerImgUrl]>">
+      <img ng-src="<[$ctrl.headerImgUrl]>" alt="Learning with Oppia">
```

Comment:
Should we move the image alt to `I18N` (Ditto elsewhere in this file)

Reply:
Fixed!